lib/chains/elementChain.js

```diff
@@ -50,12 +50,30 @@ const {
 	applyCountAndDelayAndClicks,
 	applyUntilCondition,
 } = require('../utils/socketChainHelper');
-const {PROP_COMPARATOR: PROP_COMPARATOR_M, ELEMENT_PROP, SUBJ_COMPARATOR: SUBJ_COMPARATOR_M, ELEMENT_VALUES} = require('../mappings');
+const {
+	PROP_COMPARATOR: PROP_COMPARATOR_M,
+	ELEMENT_PROP,
+	SUBJ_COMPARATOR: SUBJ_COMPARATOR_M,
+	ELEMENT_VALUES,
+} = require('../mappings');
 const {VALUE} = require('../constants/element');
 const logger = require('../utils/logger');
 
 const toString = data => {
-	const elName = data.selector.apiId || data.selector.css || data.selector.xpath || (data.selector.video && 'video');
+	let elName;
+
+	switch (true) {
+		case !!data.selector.apiId:
+			elName = data.selector.apiId + ' ';
+			break;
+		case data.selector.video && Object.keys(data.selector).length === 1:
+			elName = 'video ';
+			break;
+		default:
+			elName = '';
```

Comment:
So what about element selected by text or by attributes is this accounted for?

Reply:
selectors list is added to error message in socketErrorMessages.js line 211